lambda/outputs.tf

```diff
@@ -43,7 +43,7 @@ output "create_keycloak_user_lambda_security_group_new" {
 }
 
 output "antivirus_lambda_role" {
-  value = aws_iam_role.lambda_iam_role.*.arn
+  value = var.lambda_yara_av == true ? aws_iam_role.lambda_iam_role[0].arn : ""
```

Comment:
Being a bit dense, but what is this change for?

Reply:
We don't need it. I thought terraform would complain if I did `antivirus_lambda_role[0]` in the root.tf of terraform environments in case it didn't exist but it seems to work and I think even in a new environment, it knows enough to create this first before the endpoint so I've reverted it to how it was.